examples/quarkus/src/main/java/org/acme/OpenAPIJsonFilter.java

```diff
@@ -0,0 +1,54 @@
+package org.acme;
+
+import io.quarkus.smallrye.openapi.OpenApiFilter;
+import jakarta.ws.rs.core.MediaType;
+import org.eclipse.microprofile.openapi.OASFactory;
+import org.eclipse.microprofile.openapi.OASFilter;
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.eclipse.microprofile.openapi.models.Paths;
+import org.eclipse.microprofile.openapi.models.headers.Header;
+import org.eclipse.microprofile.openapi.models.media.Content;
+import org.eclipse.microprofile.openapi.models.media.Schema;
+import org.eclipse.microprofile.openapi.models.responses.APIResponses;
+
+@OpenApiFilter(OpenApiFilter.RunStage.BUILD)
+public class OpenAPIJsonFilter implements OASFilter {
```

Comment:
Deze implementatie is niet meer nodig als https://github.com/quarkusio/quarkus/pull/47290 gepubliceerd wordt in Quarkus